<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CallerWidget</class>
 <widget class="QWidget" name="CallerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>140</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>140</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="AcceptButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>75</width>
     <height>64</height>
    </rect>
   </property>
   <property name="text">
    <string>Accept</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DeclineButton">
   <property name="geometry">
    <rect>
     <x>135</x>
     <y>60</y>
     <width>75</width>
     <height>66</height>
    </rect>
   </property>
   <property name="text">
    <string>Decline</string>
   </property>
  </widget>
  <widget class="QLabel" name="CallerLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
